saviSimulation/savi/jason_processing/Geometry.java

```diff
@@ -82,4 +78,25 @@ public static PVector absolutePositionFromPolar(double azimuth, double elevation
 		PVector absolutePosition = new PVector(x,y,z);
 		return absolutePosition;
 	}
-}
+
+	public static double normalize02PI(double angle) {
+		while (angle < 0)                                                                                    
+			angle += (2 * Math.PI);
+		
+		while (angle > (2 * Math.PI))
+			angle -= (2 * Math.PI);
+	
+		return angle;
+	}
+
+	public static double normalize0PI(double angle) {
+		while (angle < 0)                                                                                    
+			angle += (2 * Math.PI);
```

Comment:
This method (normalize0PI) is identical to the other method (normalize02PI). I believe that this is an error on lines 94, 96, and 97. Also, I'm trying to think of when this method would be needed (but I suspect I'll see it used somewhere else in this PR).